docs/Toy/examples/ast.toy

```diff
@@ -29,47 +29,47 @@ def main() {
 }
 
 
-# CHECK: Module:
-# CHECK-NEXT:     Function
-# CHECK-NEXT:       Proto 'multiply_transpose' @{{.*}}ast.toy:4:1
-# CHECK-NEXT:       Params: [a, b]
-# CHECK-NEXT:       Block {
-# CHECK-NEXT:         Return
-# CHECK-NEXT:           BinOp: * @{{.*}}ast.toy:5:25
-# CHECK-NEXT:             Call 'transpose' [ @{{.*}}ast.toy:5:10
-# CHECK-NEXT:               var: a @{{.*}}ast.toy:5:20
-# CHECK-NEXT:             ]
-# CHECK-NEXT:             Call 'transpose' [ @{{.*}}ast.toy:5:25
-# CHECK-NEXT:               var: b @{{.*}}ast.toy:5:35
-# CHECK-NEXT:             ]
-# CHECK-NEXT:       } // Block
-# CHECK-NEXT:     Function
-# CHECK-NEXT:       Proto 'main' @{{.*}}ast.toy:8:1
-# CHECK-NEXT:       Params: []
-# CHECK-NEXT:       Block {
-# CHECK-NEXT:         VarDecl a<> @{{.*}}ast.toy:11:3
-# CHECK-NEXT:           Literal: <2, 3>[ <3>[ 1.000000e+00, 2.000000e+00, 3.000000e+00], <3>[ 4.000000e+00, 5.000000e+00, 6.000000e+00]] @{{.*}}ast.toy:11:11
-# CHECK-NEXT:         VarDecl b<2, 3> @{{.*}}ast.toy:15:3
-# CHECK-NEXT:           Literal: <6>[ 1.000000e+00, 2.000000e+00, 3.000000e+00, 4.000000e+00, 5.000000e+00, 6.000000e+00] @{{.*}}ast.toy:15:17
-# CHECK-NEXT:         VarDecl c<> @{{.*}}ast.toy:19:3
-# CHECK-NEXT:           Call 'multiply_transpose' [ @{{.*}}ast.toy:19:11
-# CHECK-NEXT:             var: a @{{.*}}ast.toy:19:30
-# CHECK-NEXT:             var: b @{{.*}}ast.toy:19:33
+#      CHECK: Module:
+# CHECK-NEXT:   Function
```

Comment:
Why had I never thought of aligning the check statements like this.

Reply:
IKR, I actually have a local branch that makes all XDSL_ROUNDTRIP strict on whitespace and full line matches and it caught some bugs, I haven't ported all the files, though. This is deep side quest territory for now.